pkg/odo/cli/pipelines/init.go

```diff
@@ -93,21 +93,23 @@ func NewCmdInit(name, fullName string) *cobra.Command {
 	}
 
 	addInitCommands(initCmd, o.InitOptions)
+
+	initCmd.MarkFlagRequired("gitops-repo-url")
```

Comment:
This is already marked as required https://github.com/rhd-gitops-example/odo/pull/157/files#diff-b14a2c841122a2c26a96c625e4f1f82eR114